src/components/docs-version-switcher/index.tsx

```diff
@@ -25,9 +25,6 @@ const DocsVersionSwitcher = ({ options }: DocsVersionSwitcherProps) => {
 
   // Don't do anything if there aren't any options
   if (!options || options.length === 0) {
-    if (IS_DEV) {
-      console.warn('DocsVersionSwitcher has no `options` to render.')
-    }
```

Comment:
Could we instead conditionally render `DocsVersionSwitcher` so this warning isn't happening on all pages? The intent behind this message is to help while debugging in case the switcher is expected to be rendered but isn't.

Reply:
I think what I was picturing is a check if `versions` itself is truthy, then render `DocsVersionSwitcher`, and then this check would be updated to check the length and log the warning. I'd see the warning being helpful in cases like when an unexpected empty array is returned from the function that fetches the versions array.

Reply:
Makes sense, will update.